### PR TITLE
add version information

### PR DESCRIPTION
**Pull request by Furtif**
add package version info 
image

### CODE REVIEW

**Comment by jfberry** [comment]
@na-ji one for you to review pls

File: package.json [hunk 1]
```diff
@@ -15,6 +15,9 @@
     "dev:mock": "NODE_ENV=development NX_ENABLE_MOCKS=1 nx run-many --target=dev --all",
     "clear-cache": "nx clear-cache"
   },
+  "exports": {
+    "./package.json": "./package.json"
+  },
```
Comment (na-ji) [inline package.json]: Why do you need to export the package.json?

Comment (Furtif) [reply package.json]: I always used it like this so I could use it as a module

Comment (na-ji) [reply package.json]: I don't see the point, though. In this case, it's not useful at all.

Comment (Fabio1988) [reply package.json]:
```suggestion
```

Comment (Furtif) [reply package.json]: > 

acepted

**Review by na-ji — APPROVED** [review]

**Comment by na-ji** [comment]
Thanks for the PR

**Pull request merged**